ci/test/cargo-test/image/run-tests

```diff
@@ -19,8 +19,7 @@ set -euo pipefail
 ci_init
 
 while read -r binary cwd; do
-    cd "$cwd"
-    ci_try "/tests/$binary" --nocapture
+    (cd "$cwd" && ci_try "/tests/$binary" --nocapture -Z unstable-options --format json --report-time | tee results.json)
     cd "$OLDPWD"
```

Comment:
So close! I think the latest test failure is resolved if you delete this line (`cd $OLDPWD`).

Reply:
> Oh apologies, I typo'd! Phrasing the command like this should ensure the `results.json` file gets emitted in the root of the repo:
> 
> ```
> (cd "$cwd" && ci_try "/tests/$binary" --nocapture -Z unstable-options --format json --report-time) | tee -a results.json
> ```

I ended up doing a slightly different route and using $MZ_ROOT for explicitness to make life easier for the next person needing to work on this